src/allocTracer.cpp

```diff
@@ -71,6 +71,9 @@ void AllocTracer::recordAllocation(void* ucontext, EventType event_type, uintptr
 
 Error AllocTracer::check(Arguments& args) {
     if (args._live) {
+        // This engine is only going to be selected in Profiler::selectAllocEngine
+        // when can_generate_sampled_object_alloc_events is not available, i.e.
+        // JDK<11.
```

Comment:
Just came across this yesterday: https://lkml.org/lkml/2020/5/29/1038
I'm fully with Linus on this issue.

Reply:
045908b9001ccfd87ecc3e55ce82136f0cb5eabb

Reply:
From one extreme to another :)
There are no strict requirements how long a line should be. Just keep it in a shape that looks fine and reads fine.
```
        // This engine is only going to be selected in Profiler::selectAllocEngine
        // when can_generate_sampled_object_alloc_events is not available, i.e. JDK<11.
```

Reply:
e42181b113f763ce3a1c6104e0abde808f93059a